components/contributors.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import glamorous, {Div} from 'glamorous'
+import {Anchor} from '../components/styled-links'
+
+const Container = glamorous.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  '& > h3': {
+    margin: '20px',
+  },
+})
+
+const Img = glamorous.img({
+  height: 45,
+  width: 45,
+  borderRadius: '100%',
+  margin: '0 5px',
+  transition: 'transform .3s',
+  ':hover': {
```

Comment:
Ugh so was close... I tried `':focus, :hover, :active', dump mistake! Thanks for the help